app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/FlankerStimulus.java

```diff
@@ -45,4 +48,13 @@ public String asText() {
     Log.d("MINT", "Stimulus array: " + stimulusArray.toString());
     return stimulusArray;
   }
+
+
+  private static final List<FlankerStimulus> VALUES = Collections.unmodifiableList(Arrays.asList(values()));
+  private static final int SIZE = VALUES.size();
+  private static final Random RANDOM = new Random();
+
+  public static FlankerStimulus randomStimulus()  {
+    return VALUES.get(RANDOM.nextInt(SIZE));
```

Comment:
I'm not sure this is how we want to do this - as we might e.g. get the the same stimulus twice.
The createStimuli method seems fine - building them all, but then randomly shuffling so that the *order* is random (rather than the number of times each is used).

Reply:
This is actually only used to randomize certain entry in the stimulusCueArray, for achieving the uncertainty in the cues. But the naming did cause some confusion, so I ll rename it.